src/pages/CSSocietyPage.js

```diff
@@ -109,13 +113,7 @@ export default function CSSocietyPage(props) {
         <Paper className={classes.paper}>
           <Typography variant="h3">Executive Committee</Typography>
           <br />
-          <Grid container spacing={2} justify="center">
-            {execom.compsoc.map(member => (
-              <Grid item xs={12} md={4}>
-                <Avatar name={member.name} position={member.position} src={member.image} />
-              </Grid>
-            ))}
-          </Grid>
+          <SocietyExecom sid = {ecats.compsoc} exelist = {execom.compsoc} />
```

Comment:
Actually, instead of fetching and sending the execom list from the parent components, let's fetch it in the `SocietyExecom` component, anyway we are passing the Sid so it should be fine to fetch and display it @naveenbteli please look into this.